shared/uavobjectdefinition/loggingsettings.xml

```diff
@@ -1,6 +1,7 @@
 <xml>
 	<object name="LoggingSettings" singleinstance="true" settings="true">
 		<description>Settings for the logging module</description>
+		<field name="LogDestination" units="" type="enum" options="SPIFlash,OpenLog" elements="1" defaultvalue="OpenLog"/>
```

Comment:
> Any board can log to the OpenLog, but only some boards have the SPI Flash chip, so OpenLog seemed to be the logical choice for a default.

I should have caught this. We have an existing base of users who are used to this working with flash chips. Most TL boards have flash chips, so while there's a certain amount of logic to this stance, it does switch away from a known-good default to one which potentially will cause (and has caused?) issues down the road.